### PR TITLE
Add: Static pages 

### LINKED ISSUE #48: Add: about us page
Hi @KeenWarrior @kunal-kushwaha ,
We should add:

- About us page
- Terms & condition
- Privacy policy

Can I work on them.

### PR DESCRIPTION
**Pull request by rajatgupta24**
**Issue Number**

fixes #48

**Describe the changes you've made**
Created DocsLayout & document folders
Now we can convert .md file to .jsx.

**Describe if there is any unusual behavior (Any Warning) of your code(Write `NA` if there isn't)**

- The new routes I made are now showing beneath the Homepage, which wasn't happening before. 

**Additional context (OPTIONAL)**
In prev PR there were some conflicts in the package-lock.json file.
There were some wrong packages that were installed,
You can see that the right code is having half no. of lines than the prev one.

**Test plan (OPTIONAL)**

A good test plan should give instructions that someone else can easily follow.

**Checklist**

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] The title of my pull request is a short description of the requested changes.

### CODE REVIEW

**Comment by rajatgupta24** [comment]
@Abhishek-kumar09 can you please help me figure out what's causing this thing in routes.js 
image
The /privacy and /terms-and-conditions and now rendering beneath home-page.

File: src/Routes.js [hunk 1]
```diff
@@ -24,6 +27,10 @@ const renderRoutes = () => (
         <Route path="/profile" exact component={ProfilePage} />
       </SwipeableRoutes>
     </MainLayout>
+    <DocsLayout>
```
Comment (Abhishek-kumar09) [inline src/Routes.js:30]: > The /privacy and /terms-and-conditions and now rendering beneath home-page.

Add it in the switch instead.

I have added switch in the latest code of main branch please see it.

**Comment by rajatgupta24** [comment]
Hi @Abhishek-kumar09, these are some screenshots of the changes I made. (FAQs, About Us, Terms & Conditions, Privacy Policy)

image 
image 
image

**Comment by rajatgupta24** [comment]
Hi @Abhishek-kumar09, there are still few changes left to make regarding this topic, so should I just continue pushing code in this PR or create a new one.

**Pull request merged**